### PR TITLE
Improvement / HOLO 1532 Review return values for contracts

### PR DESCRIPTION
**Pull request by BravoNatalie**
## Describe Changes

Changes made to the package: **sdk**.

This PR addresses the need to ensure type safety by adjusting the return types in our contract functions.

- Updated the return types for several contract functions to align with our type definitions.
- Modified the associated tests to reflect the changes in return types.
- Add new Operator contract functions

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All tests are passing

### CODE REVIEW

File: packages/sdk/src/contracts/factory.contract.ts [hunk 1]
```diff
@@ -232,15 +250,9 @@ export class FactoryContract extends HolographBaseContract {
...
    * @param payload The payload of the request.
-   * @returns The function selector, and its calldata.
+   * @returns An array with the bridgeOut function selector and the provided payload
    */
-  async bridgeOut(
-    chainId: number,
-    toChain: number,
-    sender: Address,
-    payload: Hex,
-    wallet?: {account: string | HolographWallet},
```
Comment (feliperodriguess) [inline packages/sdk/src/contracts/factory.contract.ts:242]: why did you remove the `wallet` param?

**Review by feliperodriguess — APPROVED** [review]

File: packages/sdk/src/contracts/factory.contract.ts (hunk 1 above)
Comment (BravoNatalie) [reply packages/sdk/src/contracts/factory.contract.ts:242]: Since this is a pure function in the contract, it doesn't alter the state and thus does not generate a transaction.
image

**Review by sogoiii — APPROVED** [review]
👍
